.github/actions/deploy-frontend/action.yml

```diff
@@ -0,0 +1,40 @@
+# action.yml
+name: 'Upload Static Site (React)'
+description: 'Push react website to Azure blob storage'
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: Set environment variables - dev
+      if: github.ref == 'refs/heads/dev-rheft'
+      shell: bash
+      run: echo "IMAGE_REPO=rkh5012" >> $GITHUB_ENV
+
+    - name: Set environment variables - test
+      if: github.ref == 'refs/heads/ronheft/test-investigation'
+      shell: bash
+      run: echo "IMAGE_REPO=pdhtest" >> $GITHUB_ENV
+
+    - name: Set environment variables - staging
+      if: github.ref == 'refs/heads/master'
+      shell: bash
+      run: echo "IMAGE_REPO=pdhstaging" >> $GITHUB_ENV
+
+    - name: Set environment variables - production
+      if: github.ref == 'refs/heads/production'
+      shell: bash
+      run: echo "IMAGE_REPO=pdhprod" >> $GITHUB_ENV
+
+    - name: Download Frontend Artifact (React)
```

Comment:
Comment only. It should be noted that there's no "else if" for setting env variables when it isn't one of the cases above (e.g just a regular PR where the actions run). 

I'm not even sure there is a way to do else-if... maybe setting some git-action-variable as a side effect when there's a match and then an `if` that checks it's been set?

I don't think it matters but just trying to think through what might happen if `IMAGE_REPO` is unset.

The 
``` bash
az storage blob delete-batch --account-name ${{ env.IMAGE_REPO }}public -s '$web'
az storage blob upload-batch --account-name ${{ env.IMAGE_REPO }}public -s frontend-react/build -d '$web'
  ```
becomes 
``` bash
az storage blob delete-batch --account-name public -s '$web'
az storage blob upload-batch --account-name public -s frontend-react/build -d '$web'
```

It seems like this is what the old code did? (This is not blocking this PR)

Reply:
Good point. We need defaults and/or prevent deploy if variables are not set.

Reply:
Added conditional line:36